lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -158,6 +158,7 @@ export const migrateFileRecord = async (
     file_name: file.fileName,
     source: file.source,
     path: file.path,
+    type: file.type,
```

Comment:
So is the intent that we're going to ask people to re-run their data-migration2 in order to populate the `type` field of their file records? If so, that seems like it could be problematic for a few reasons

Reply:
The intent here is post migration the tool uses the new field, as we're not removing it.     There's a fair question of if we should rip it out entirely, but if we're going to maintain and release it with this work shouldn't itdo the right thing relative to the work?

Reply:
I'm not sure if it will work because of the inherent "skip" logic built into the migration: it only updates records in PG when the Dynamo record is newer. However, we want to update all PG granule files with the `type` field from the Dynamo granule files